### ISSUE TITLE
question about batch size and learnrate

### ISSUE THREAD

**Issue by gallenszl** [DeepMotionAIResearch/DenseMatchingBenchmark#7]
Hello, I want to know that in the paper you set batch size to 3 for training on 3 1080TI with learning rate 0.001 in seceneflow,but in the README you said that you use 8 GPUS to train(batch size 16) with learning rate 0.001,so which one did you get the best result in kitti? 
Thanks

**Comment by youmi-zym** [comment]
Hi, @gallenszl, 
  The setting in our paper is quite different from the setting in this new architecture.  But it's fine as long as you train our model with lr=0.001 and GPUs=3 or 4, as more GPUs will lead to fewer iterations. 
To be clear,  I detail the training schedule in this framework here:
GPUs <= 4, we find GPUs=4 works well in this framework.
batch_size=1
For SceneFlow, training with constant lr=0.001 for 20 epochs;
For KITTI, training with start lr=0.001 for 600 epochs, and decay 1/3 at epochs=100, 300.
I hope this information helps you, and I'll release the checkpoint this week for your reimplementation.

**Comment by youmi-zym** [comment]
Hi, @gallenszl 
I'm sorry to tell you that the KITTI 2015 benchmark crashed, framework updating and checkpoint release are both delayed until it works normally.

**Comment by gallenszl** [comment]
Thank you for your notice.

**Comment by youmi-zym** [comment]
updated, enjoy it.
